README.md

```diff
@@ -121,20 +121,34 @@ you also save 1 RPC call.
 The profile function will always request an ETH addr record.
 For names, this means the address will always at the top level of the returned object.
 For addresses, this means the "match" property (a boolean value for matching reverse/forward resolution) will always be at the top level of the returned object.
+`coinType` is the cryptocurrency coin type index from [SLIP44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
+
 
 ```js
 /* Normal profile fetching */
 const profile = await FNSInstance.getProfile('juan.fil')
 
-/* Normal address fetching */
-const address = await FNSInstance.getAddress('juan.fil')
+/* Get Blockchain Address
+Returns the Blockchain address associated with the provided node and coinType, or 0 if none. */
+
+/* For FIL Address - t410fkyutpa243vojf5klstpwld6twufgr3gv5rmsx5y */
+const address = await FNSInstance.getAddress('juan.fil', 4610)
```

Comment:
can we map these coin type map as enum and export it to public so that user don't have to type number or need to find them